apps/token-manager/app/src/script.js

```diff
@@ -0,0 +1,166 @@
+import Aragon from '@aragon/client'
+import tokenSettings, { hasLoadedTokenSettings } from './token-settings'
+import minimeArtifact from '@aragon/os/build/contracts/MinimeToken.json'
```

Comment:
This would be much better if it were just the ABI, rather than the entire truffle artifact.

Pinging @ji to bring an `abi` export to aragonOS 😊